examples/0_Database_handling/plot_explore_database_vaex.py

```diff
@@ -16,9 +16,17 @@
 
 """
 
+try:
```

Comment:
@minouHub this was my alternative (to keep the example the same)

```
#%% Discard this example if Vaex is not installed
try:
    import vaex
except ImportError:
    quit()

# ----
```

Reply:
@minouHub it doesn't work ?

Reply:
no, the example in the doc was just stopping at that point. I guess it's because there was no plot generated for the documentation image. Anyway, it's working now for a pandas dataframe